apps/wallet-mobile/src/yoroi-wallets/cardano/assetUtils/assetUtils.ts

```diff
@@ -1,19 +1,24 @@
 import BigNumber from 'bignumber.js'
 
-import {NetworkId, RawUtxo} from '../../types/other'
+import {Address} from '../../types'
+import {RawUtxo} from '../../types/other'
 import {CardanoMobile} from '../../wallets'
-import {getCardanoNetworkConfigById} from '../networks'
-import {cardanoValueFromRemoteFormat} from '../utils'
+import {COINS_PER_UTXO_BYTE} from '../constants/common'
+import {cardanoValueFromRemoteFormat, normalizeToAddress} from '../utils'
 
-export async function calcLockedDeposit(utxos: RawUtxo[], networkId: NetworkId) {
-  const networkConfig = getCardanoNetworkConfigById(networkId)
-  const coinsPerUtxoWord = await CardanoMobile.BigNum.fromStr(networkConfig.COINS_PER_UTXO_WORD)
+export async function calcLockedDeposit(utxos: RawUtxo[], address: Address, coinsPerUtxoByteStr: string) {
   const utxosWithAssets = utxos.filter((u) => u.assets.length > 0)
 
+  const coinsPerUtxoByte = await CardanoMobile.BigNum.fromStr(coinsPerUtxoByteStr ?? COINS_PER_UTXO_BYTE)
+  const dataCost = await CardanoMobile.DataCost.newCoinsPerByte(coinsPerUtxoByte)
+  const normalizedAddress = await normalizeToAddress(address)
```

Comment:
@jorbuedo is it real address of utxo ?

Reply:
It's a `wallet.receiveAddresses[0]`. From what I understood, I needed a wallet's address for the next operations?

Reply:
@jorbuedo it depends where data come from, because if it is change utxo ( I assume `wallet.receiveAddresses[0] ` is change address ) it is okay. But if you are trying to calc min ada for utxo that you receives from API, you need to use address of utxo not just wallet.receiveAddresses[0].
But there is also trick, if you are sure that all utxos have a BaseAddress , the `wallet.receiveAddresses[0]`  will work. 
I guess if it non standard wallet the `wallet.receiveAddresses[0]`  will work but keep in mind that there is this edge case.